tests/Unit/UseCases/CreatePayment/CreatePaymentUseCaseBuilder.php

```diff
@@ -119,4 +125,17 @@ public function withUrlGeneratorFactory( UrlGeneratorFactory $factory ): self {
 	private function makePaymentValidator(): PaymentValidator {
 		return new PaymentValidator();
 	}
+
+	private function makeFailingBankDataGenerator(): BankDataGenerator {
+		return new class implements BankDataGenerator {
```

Comment:
TIL, is this returning an anonymous object? I'm guessing a stdClass that also implements the interface.

Reply:
Found it in the docs https://www.php.net/manual/en/language.oop5.anonymous.php